From: Shawn Landden <shawnlandden@gmail.com>
Date: Sat, 7 Jul 2012 18:14:04 -0700
Subject: doc: kyotoproducts.pdf

fix link to kyotoproducts.pdf to point to Internet

There IS an old version of this presentation in the old 1.1.1 release of
Kyotocabinet under the GPL.[1] It comes with the "preferred form of
modification" source .ppt, which opens with LibreOffice. I did not
figure out what font I needed to install to get the Japaneese on the
last slide to show in LibreOffice.

Anyways, at this time, I'd rather not ship the old pdf.

[1]http://fallabs.com/kyotocabinet/pkg/old/kyotocabinet-1.1.1.tar.gz
---
 doc/index.html |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/index.html b/doc/index.html
index 17ff4bf..66da1dd 100644
--- a/doc/index.html
+++ b/doc/index.html
@@ -71,7 +71,7 @@
 <ul>
 <li><a href="spex.html">Fundamental Specifications</a></li>
 <li><a href="command.html">Specifications of Command Line Utilities</a></li>
-<li><a href="kyotoproducts.pdf">Presentation</a></li>
+<li><a href="http://fallabs.com/kyotocabinet/kyotoproducts.pdf">Presentation</a></li>
 <li><a href="api/index.html">API Documents of the core library (C/C++)</a></li>
 </ul>
 
-- 
